geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementResult.java

```diff
@@ -112,7 +116,8 @@ public void setUri(String uri) {
 
   @JsonIgnore
   public boolean isSuccessful() {
-    return statusCode == StatusCode.OK;
+    return statusCode == StatusCode.OK || statusCode == StatusCode.ACCEPTED
```

Comment:
Why are ACCEPTED and IN_PROGRESS considered successful? Does "isSuccessful" just mean it has not failed yet?

Reply:
isSuccessful just means not-an-error